components/eam/src/chemistry/mozart/mo_exp_sol.F90

```diff
@@ -122,16 +129,45 @@ subroutine exp_sol( base_sol, reaction_rates, het_rates, extfrc, delt, xhnm, nco
        elseif (trim(solsym(l)) == 'pom_a4' .or. trim(solsym(l)) == 'pom_a1' .or. trim(solsym(l)) == 'pom_a3') then
                ! V2-like explicit equation 
           do i = 1,ncol
-             do k = 1,pver
+             do k = 1,ltrop(i)
+             ! above tropopause, emission only
+                chem_loss(i,k,l) = 0.0_r8
+                chem_prod(i,k,l) = ind_prd(i,k,m)
+                base_sol(i,k,l) = base_sol(i,k,l) + delt * ind_prd(i,k,m)
+             end do
+             do k = ltrop(i)+1,pver ! brown carbon oxidation to form pom only in troposphere 
```

Comment:
Why do we need this for POM? Is it because of POM emissions in the stratosphere?